Makefile

```
@@ -202,7 +202,26 @@ drupalcheckval:
 	@docker run --rm -v `pwd`:`pwd` -w `pwd` skilldlabs/docker-drupal-check:last drupal-check -ad -vv -n --no-progress web/modules/custom/
 
 
-## Run all quality validations
-validation: | phpcs clang cinsp compval watchdogval drupalcheckval
-# This target is executed by scripts/git_hooks/pre-commit.sh as git pre-commit hook
+## Behat scenarios validation
+behat:
+	@echo "Getting base url"
+ifdef REVIEW_DOMAIN
+	$(eval BASE_URL := $(MAIN_DOMAIN_NAME))
```

For review apps we should assign review domain here instead of main domain
```suggestion
	$(eval BASE_URL := $(REVIEW_DOMAIN))
```

---

Gonna double check that,  thanks

---

no relevante